xbmc/addons/binary-addons/BinaryAddonManager.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ *      Copyright (C) 2005-2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Kodi; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "BinaryAddonManager.h"
+
+#include "addons/AddonManager.h"
+#include "utils/log.h"
+
+using namespace ADDON;
+
+CBinaryAddonManager::CBinaryAddonManager()
+{
+
+}
+
+CBinaryAddonManager::~CBinaryAddonManager()
+{
+
+}
+
+bool CBinaryAddonManager::Init()
+{
+  std::vector<std::pair<std::string, bool>> binaryAddonList;
+  if (!CAddonMgr::GetInstance().GetInstalledBinaryAddonPaths(binaryAddonList))
+  {
+    CLog::Log(LOGERROR, "Binary Addons: failed to get list");
```

Comment:
could you adjust the wording to follow the log "syntax" you're using everywhere else (\_\_FUNCTION\_\_ and the like)?